CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### vNEXT
 
+### v0.8.3
```

Comment:
For future PRs, please don't set the version number here and in package.json - sometimes we publish several things in one version